erpnext/manufacturing/report/work_order_summary/work_order_summary.py

```diff
@@ -58,21 +58,22 @@ def get_chart_data(data, filters):
 		return get_chart_based_on_qty(data, filters)
 
 def get_chart_based_on_status(data):
-	labels = ["Completed", "In Process", "Stopped", "Not Started"]
+	labels = ["Completed", "In Process", "Stopped", "Not Started", "Closed"]
```

Comment:
Don't hardcode this? 

Use `get_meta("Work Order").get_options("status")` to get status options.